src/statsig/index.ts

```diff
@@ -52,3 +56,14 @@ export function getDynamicConfigParams<T extends Record<string, StatsigParameter
     return defaultValues
   }
 }
+
+export async function updateStatsigUser(statsigUser?: StatsigUser) {
+  const state = store.getState()
+  const defaultUser = {
+    userID: walletAddressSelector(state),
+    custom: {
+      startOnboardingTime: startOnboardingTimeSelector(state),
+    },
+  }
+  await Statsig.updateUser(_.merge(defaultUser, statsigUser))
```

Comment:
does this do a deep merge? Can you add a test case for when custom includes properties but `startOnboardingTime` isn't one of them? We don't have a use case now, but in the future there could be a case where the default user includes multiple custom properties and `updateStatsigUser` is invoked with a subset of custom properties

Reply:
yeah it does do a deep merge -- added a test to explicitly test for this